lambo

```diff
@@ -175,6 +179,8 @@ AUTH=false
 NODE=false
 CODEEDITOR=""
 BROWSER=""
+DB_USERNAME="root"
+DB_PASSWORD="secret"
```

Comment:
The default will need to be an empty string.

Reply:
changed